[2023-09-24T02:01:26.510+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: create_tables.create_users_table scheduled__2023-09-23T00:00:00+00:00 [queued]>
[2023-09-24T02:01:26.533+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: create_tables.create_users_table scheduled__2023-09-23T00:00:00+00:00 [queued]>
[2023-09-24T02:01:26.534+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-09-24T02:01:26.535+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-09-24T02:01:26.536+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-09-24T02:01:26.564+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_users_table> on 2023-09-23 00:00:00+00:00
[2023-09-24T02:01:26.573+0000] {standard_task_runner.py:55} INFO - Started process 77 to run task
[2023-09-24T02:01:26.579+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'create_tables', 'create_users_table', 'scheduled__2023-09-23T00:00:00+00:00', '--job-id', '93', '--raw', '--subdir', 'DAGS_FOLDER/process_data.py', '--cfg-path', '/tmp/tmp7mohonmg']
[2023-09-24T02:01:26.580+0000] {standard_task_runner.py:83} INFO - Job 93: Subtask create_users_table
[2023-09-24T02:01:26.753+0000] {task_command.py:376} INFO - Running <TaskInstance: create_tables.create_users_table scheduled__2023-09-23T00:00:00+00:00 [running]> on host 20e0c20d2e1d
[2023-09-24T02:01:26.921+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=create_tables
AIRFLOW_CTX_TASK_ID=create_users_table
AIRFLOW_CTX_EXECUTION_DATE=2023-09-23T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-23T00:00:00+00:00
[2023-09-24T02:01:26.937+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-09-24T02:01:26.944+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
                first_name TEXT NULL,
                last_name TEXT NULL,
                email TEXT NULL,
                street_number SMALLINT NULL,
                street_name TEXT NULL,
                surburb TEXT NULL,
                post_code INTEGER NULL,
                state TEXT NULL,
                country TEXT NULL
            );
        , parameters: None
[2023-09-24T02:01:26.946+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-09-24T02:01:26.968+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=create_tables, task_id=create_users_table, execution_date=20230923T000000, start_date=20230924T020126, end_date=20230924T020126
[2023-09-24T02:01:27.032+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-09-24T02:01:27.126+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
